spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -83,6 +83,23 @@ module XCPretty
 )
     end
 
+    it "formats uncaught exceptions" do
+      @formatter.format_uncaught_exception("testRaisingUncaughtException",
+      'NSInvalidArgumentException',
+      '-[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]',
+      ['	0   CoreFoundation                      0x0000000109479a75 __exceptionPreprocess + 165',
+       '	1   libobjc.A.dylib                     0x0000000109112bb7 objc_exception_throw + 45',
+       '	2   CoreFoundation                      0x000000010938503f -[__NSPlaceholderDictionary initWithObjects:forKeys:count:] + 383',
+       '	3   CoreFoundation                      0x0000000109397d8b +[NSDictionary dictionaryWithObjects:forKeys:count:] + 59']).should == %Q(
```

Comment:
Line is too long. [142/80]<br>Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.